Topic 1/src/point7/PlayListState.java

```diff
@@ -0,0 +1,13 @@
+package point7;
+
+public interface PlayListState {
+	/**
+	 * 
+	 * @author cris14
+	 * State Pattern
+	 *
+	 */
+	public void doAction(Context context);
```

Comment:
[High] ]You should have all possible actions.
Remember that you have to perform the action, not only depending on that action but also from current status (that's why you need the context).